src/gfn/losses/flow_matching.py

```diff
@@ -20,8 +20,19 @@ class FMParametrization(Parametrization):
     to $\mathbb{R}^+$. Which is equivalent to the set of functions from the internal nodes
     (i.e. without $s_f$) to $(\mathbb{R})^{n_actions}$, without the exit action (No need for
     positivity if we parametrize log-flows).
+
+    The loss is described in section
+    3.2 of [GFlowNet Foundations](https://arxiv.org/abs/2111.09266).
+
+    Attributes:
+        logF: LogEdgeFlowEstimator
+        alpha: weight for the reward matching loss.
     """
     logF: LogEdgeFlowEstimator
+    alpha: float = 1.0
+
+    def __post_init__(self) -> None:
+        self.env = self.logF.env
```

Comment:
Just another comment with no action required - I'm definitely going to try to figure out how to *not* pass the `env` everywhere in a follow up PR after the release.